### DISCUSSION TITLE
Simplified interface options?

### DISCUSSION THREAD

**Discussion by ghost** [oobabooga/text-generation-webui#5512] [General]
Now, don't misunderstand, i personally love the default interface and all its options, but its overwhelming for a newbie.

What i would like to do is offer this to a couple of people that have no clue what 99% of this stuff means. My wish to do  (  and i know some of this is already possible and i do it, just listing it anyway to be complete of my goal )

    Load a predefined model on start ( and will be running as a service )
    Load it with all the prompt and general presets that i want including things like GPU split and ram use, tokens, temp, etc.
    Load it in multi user mode ( only a couple would have access, but still this would be the better mode to use ) 
    load it in chat instruct mode

( now the hard stuff )

        Basically the stuff on the 'chat' tab is fine.
        Still give them option to switch to regular chat, or instruct. Let them do the normal chat stuff like new chat, continue, save, rename, etc.
       BUT i want hide all the other tabs so they cant go to notebook or default mode, mess with parameters, try to load a new model, get to the training, etc.  Just one tab is all they need. ( and easier than saying 'dont go there'.. if its hidden ) 

So basically an 'admin' view and a 'user' view. And i know i could do this by changing the code, but i really don't want to go that far and have to redo it on every release or risk general breakage.

( and if this is doable, and documented somewhere, i missed it so i apologize up front )  EDIT: and sorry for formatting, its not cooperating for me. grrrr  )

**Comment by pay-tracker** [comment]
Did you ever find a solution to this?
